### ISSUE TITLE
Display the category DNSSEC in grey when the DNSSEC is not used

### ISSUE THREAD

**Issue by mtoma** [dotse/zonemaster-gui#155]
Display the category DNSSEC in grey when the DNSSEC is not used

**Comment by mtoma** [comment]
Needs a very clear explanation of how to determine that the DNSSEC is not used.

**Comment by pawal** [comment]
The NEITHER_DNSKEY_NOR_DS can be used to determine that neither record is present.

**Comment by pamasse** [comment]
Create a ticket in Zonemaster.

**Comment by matsduf** [comment]
This issue is larger than GUI. A new ticket is created in Zonemaster, https://github.com/dotse/zonemaster/issues/538

**Comment by pamasse** [comment]
@matsduf It is still relevant? It's more a profil "issue" no ?

**Comment by matsduf** [comment]
Yes, this is still relevant. It is not a profile issue, not "profile" as we have discussed profile, at least.
